README.md

```diff
@@ -29,13 +29,14 @@ extensions useful:
 
 The challenges assume three files (replace 00 with the day of the challenge).
 
-- `Data/Day00.txt`: the input data provided for the challenge
+- `Sources/Data/Day00.txt`: the input data provided for the challenge
 - `Sources/Day00.swift`: the code to solve the challenge
 - `Tests/Day00.swift`: any unit tests that you want to include
 
-To start a new day's challenge, make a copy of these files and update as
-necessary. The `AdventOfCode.swift` file controls which day's challenge is run
-with `swift run`. By default that runs the most recent challenge in the package.
+To start a new day's challenge, make a copy of these files, updating 00 to the day number,
```

Comment:
It would be good to include a diff style code block:
```diff
// Add each new day implementation to this array:
let allChallenges: [any AdventDay] = [
-  Day00()
+  Day00(),
+  Day01(),
]
```

Reply:
Wow -- I never knew that the color-coding came through in Markdown. Neat!